zh/access-tidb.md

```diff
@@ -4,46 +4,42 @@ summary: 介绍如何访问 Kubernetes 上的 TiDB 集群。
 aliases: ['/docs-cn/tidb-in-kubernetes/stable/access-tidb/','/docs-cn/tidb-in-kubernetes/v1.1/access-tidb/','/docs-cn/dev/how-to/deploy/orchestrated/tidb-in-kubernetes/access-tidb/','/docs-cn/v3.0/how-to/deploy/orchestrated/tidb-in-kubernetes/access-tidb/']
 ---
 
-# 访问 Kubernetes 上的 TiDB 集群
+# 访问 TiDB 集群
 
-在 Kubernetes 集群内访问 TiDB 时，使用 TiDB service 域名 `${cluster_name}-tidb.${namespace}` 即可。
+Service 可以根据场景配置不同的类型，比如 `ClusterIP`、`NodePort`、`LoadBalancer` 等，对于不同的类型可以有不同的访问方式。
 
-若需要在集群外访问，则需将 TiDB 服务端口暴露出去。在 `TidbCluster` CR 中，通过 `spec.tidb.service` 字段进行配置：
+可以通过如下命令获取 TiDB Service 信息：
 
-{{< copyable "" >}}
+{{< copyable "shell-regular" >}}
 
-```yaml
-spec:
-  ...
-  tidb:
-    service:
-      type: NodePort
-      # externalTrafficPolicy: Cluster
-      # annotations:
-      #   cloud.google.com/load-balancer-type: Internal
+```shell
+kubectl get svc ${serviceName} -n ${namespace}
 ```
 
-> **注意：**
->
-> [MySQL 8.0 默认认证插件](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_default_authentication_plugin)从 `mysql_native_password` 更新为 `caching_sha2_password`，因此如果使用 MySQL 8.0 客户端访问 TiDB 服务（TiDB 版本 < v4.0.7），并且用户账户有配置密码，需要显示指定 `--default-auth=mysql_native_password` 参数。
+示例：
 
-## NodePort
+```
+# kubectl get svc basic-tidb -n default
+NAME         TYPE       CLUSTER-IP     EXTERNAL-IP   PORT(S)                          AGE
+basic-tidb   NodePort   10.233.6.240   <none>        4000:32498/TCP,10080:30171/TCP   61d
+```
 
-在没有 LoadBalancer 时，可选择通过 NodePort 暴露。NodePort 有两种模式：
+上述示例描述了 `default` namespace 下 `basic-tidb` 服务的信息，类型为 `NodePort`，ClusterIP 为 `10.233.6.240`，ServicePort 为 `4000` 和 `10080`，对应的 NodePort 分别为 `32498` 和 `30171`。
 
-- `externalTrafficPolicy=Cluster`：集群所有的机器都会给 TiDB 分配 TCP 端口，此为默认值
+> **注意：**
+>
+> [MySQL 8.0 默认认证插件](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_default_authentication_plugin)从 `mysql_native_password` 更新为 `caching_sha2_password`，因此如果使用 MySQL 8.0 客户端访问 TiDB 服务（TiDB 版本 < v4.0.7），并且用户账户有配置密码，需要显示指定 `--default-auth=mysql_native_password` 参数。
 
-    使用 `Cluster` 模式时，可以通过任意一台机器的 IP 加同一个端口访问 TiDB 服务，如果该机器上没有 TiDB Pod，则相应请求会转发到有 TiDB Pod 的机器上。
+## ClusterIP
 
-    > **注意：**
-    >
-    > 该模式下 TiDB 服务获取到的请求源 IP 是主机 IP，并不是真正的客户端源 IP，所以基于客户端源 IP 的访问权限控制在该模式下不可用。
+`ClusterIP` 是通过集群的内部 IP 暴露服务，选择该类型的服务时，只能在集群内部访问，可以通过如下方式访问：
 
-- `externalTrafficPolicy=Local`：只有运行 TiDB 的机器会分配 TCP 端口，用于访问本地的 TiDB 实例
+* ClusterIP + ServicePort
+* Service 域名 (`${serviceName}.${namespace}`) + ServicePort
```

Comment:
Should we insert a ":" between IP and Port?

Reply:
The host and port are used separately when used to connect to TiDB service, so it should not be necessary to have `:` here, right?